### ISSUE TITLE
Consider exposing the XRReflectionProbe off the XRSession object

### ISSUE THREAD

**Issue by kearwood** [immersive-web/lighting-estimation#25]
As the `XRReflectionProbe`'s cube map can likely be used for multiple frames and is not updated every frame, perhaps it should be exposed on the `XRSession` object rather than `XRFrame`.  It should also have an event to indicate when a new cube map is available.

The WebGL rendering engine could then interpolate between the last and new cube map when a new one is available.

**Comment by mounirlamouri** [comment]
Given that it could be updated at most every frame, wouldn't it make sense to have it hanging on `XRFrame` and make sure the developer is aware/can figure out if the frame has changed?

**Comment by blairmacintyre** [comment]
I agree, I would think it would be in the frame.  During the discussion of hit test, and real world geometry, we talked about having a single data structure for "real world info" that any information about the world (hit tests, geometry, illumination, and so on) would be part of (to avoid littering the frame object with extra fields as features are added).

**Comment by mounirlamouri** [comment]
> During the discussion of hit test, and real world geometry, we talked about having a single data structure for "real world info" that any information about the world (hit tests, geometry, illumination, and so on) would be part of (to avoid littering the frame object with extra fields as features are added).

I think the proposal in this repo doesn't use that object. Piotr's proposal that we have behind a flag in Chrome is this though. In general, we are fine either way. It would be nice to have one object holding things but also would create cyclical dependencies between specs for something fairly minor.
